### ISSUE TITLE
Youtube + mindustry = pain

### ISSUE THREAD

**Issue by awia-the-fuwwy** [Anuken/Mindustry#9779]
### Platforms

Android

### Build

Release build 146

### Issue

The audio started cutting out and then the phone turned off, and when I tried turning it back on it either just didn't turn on, turned back off immediately after, or wouldn't let me unlock it

### Steps to reproduce

I had youtube playing picture in picture and then opened mindustry

### Save file

[uwu.zip](https://github.com/Anuken/Mindustry/files/15100009/uwu.zip)

### Submission

- [X] I have updated to the latest release (https://github.com/Anuken/Mindustry/releases) to make sure my issue has not been fixed.
- [X] I have searched the closed and open issues to make sure that this problem has not already been reported.

**Comment by Anuken** [comment]
If your phone system breaks down after launching the gone, that is a bug in your phone system, not Mindustry.

**Comment by awia-the-fuwwy** [comment]
Yeah you're right I think I just overloaded the phone by having them both open, sorry for the trouble
